src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+html {
```

Comment:
```suggestion
* {
```
`html` selector affects only for `html` tag

Reply:
I don't understand because in tasks we don't use *